### ISSUE TITLE
Nothing to "see below" for [shared_]future::get

### ISSUE THREAD

**Issue by timsong-cpp** [cplusplus/concurrency-ts#100]
The `future` synopsis in [futures.unique_future] and `shared_future` synopsis in [futures.shared_future] each depicts a `see below get();`, presumably copied from C++14.

There's nothing "below" to see for `get()`.

**Comment by jwakely** [comment]
[futures.unique_future] p13 to p18 and [futures.shared_future] p16 to p20?

**Comment by timsong-cpp** [comment]
@jwakely That's the standard. This is about the concurrency TS.

**Comment by jwakely** [comment]
Oops, sorry, I read the email and completely missed which repo this was filed for.

**Comment by arturl** [comment]
The synopsis is copied from the Standard. The TS says that the missing subclauses are the same as in the Standard:

> The specifications of all declarations within this subclause 2.3 and its subclauses are the same as the corresponding declarations, as specified in C++14 §30.6.6, unless explicitly specified otherwise

**Comment by timsong-cpp** [comment]
Well, if it's pointing to that paragraph, then it should be see "above", rather than "below", since the paragraph precedes the synopsis.

**Comment by arturl** [comment]
No, it's not pointing to that paragraph. The paragraph simply states that some clauses are assumed to exist, so the text in the synopsis refers to those hidden clauses.
